packages/cmf-webpack-plugin/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@talend/react-cmf-webpack-plugin",
+  "description": "@talend/react-cmf webpack plugin for merging CMF settings",
+  "main": "dist/index.js",
+  "scripts": {
+    "prepublish": "npm run build && npm run test",
+    "build": "rimraf ./dist && babel -d dist ./src/ --ignore **/*.test.js",
+    "start": "echo nothing to start",
+    "test": "echo nothing to test yet in @talend/react-cmf-webpack-plugin",
+    "lint:es": "eslint --config ../../.eslintrc --ext .js src",
+    "lint": "npm run lint:es"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Talend/ui.git"
+  },
+  "keywords": [
+    "react",
+    "cmf",
+    "webpack",
+    "plugin",
+    "framework"
+  ],
+  "author": "Talend Frontend <frontend@talend.com>",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/Talend/ui/issues"
+  },
+  "homepage": "https://github.com/Talend/ui/blob/master/packages/cmf/README.md",
+  "dependencies": {
+    "@talend/react-cmf": "file:../cmf",
```

Comment:
Please replace by version. It is part of yarn workspace, so the links are done automatically